### PR TITLE
add nestedjaggedarray.ipynb

### PR DESCRIPTION
**Pull request by yihui-lai**

### CODE REVIEW

**Comment by yimuchen** [comment]
Can you add the contents of the `nestedjaggedarray` notebook into the `coffeaschema` notebook? It helps keep the index page cleaner?

**Comment by yimuchen** [comment]
Have you checked if the binder configurations work? If you have, can you add a link to the index page in the second paragraph?

File: source/notebooks/higgsto4l.ipynb [hunk 1]
```diff
@@ -27,8 +27,8 @@
    "source": [
     "%matplotlib inline\n",
     "import numpy as np\n",
-    "import awkward1 as ak\n",
-    "import awkward\n",
+    "import awkward as ak\n",
+    "import awkward0\n",
```
Comment (yimuchen) [inline source/notebooks/higgsto4l.ipynb:31]: Is moving to awkward1 required for binder? If this is the case, I feel like we should prioritize keeping the interface as up to date as possible over having an binder example

Comment (yihui-lai) [reply source/notebooks/higgsto4l.ipynb:31]: Yes, this move is for binder. I find some version conflicts. ok, maybe I can close this PR.

**Pull request closed without merging**